packages/ui/src/FilesTable/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { ActionsMenu, DataTable } from "../index";
+import { FilterMatchMode } from "primereact/api";
+import { ColumnProps } from "primereact/column";
+import { MenuItem } from "primereact/menuitem";
+
+type TranslationMessage = {
+  actionDownloadMessage: string;
+  actionEditMessage: string;
+  actionRenameMessage: string;
+  columnFileHeaderMessage: string;
+  columnDescriptionHeaderMessage: string;
+  searchPlaceholderMessage: string;
+  columnUploadedByHeaderMessage: string;
+  columnUploadedAtHeaderMessage: string;
+  columnActionsHeaderMessage: string;
+  tableEmptyMessage: string;
+};
```

Comment:
Three things:
- We don't know if they are translation or just message. So just "Message" or "Messages" is enough
- Since we already saying they are messages, we don't need to write Message prefix on each attribute
- All attributes should be optional

```suggestion
type Messages = {
  downloadAction?: string;
  editDescriptionAction?: string;
  renameAction?: string;
  filenameColumnHeader?: string;
  descriptionColumnHeader?: string;
  uploadedByColumnHeader?: string;
  uploadedAtColumnHeader?: string;
   actionsColumnHeader?: string;
  searchPlaceholder?: string;
  tableEmpty?: string;
};
```